inline-caching/README.md

```diff
@@ -0,0 +1,247 @@
+3. ItemRepository - defines key and value of our data model
+4. ItemService - uses a Repository and can utilize business logic to adapt GemFire data into a nicer format 
+
+
+The web application will retrieve a request from the user, and then ask GemFire for the data in it's `item` region. 
+For read operations to the region, if the data exists in the GemFire region, the data will be retrieved without using the CacheLoader. 
+If the data does **not** exist in the GemFire region, the data will be fetched via the CacheLoader interface defined above.
+
+For write operations to the GemFire region, the data is written to the GemFire region first and then batched up to be written to postgres using the AsyncEventListener.
+
+## Start the Spring Boot web service with GemFire integration
+
+Connecting a GemFire client to an existing GemFire cluster is easy. 
+In this example, we have a simple client that passes all reads an writes to the web server. 
+The web server is a Spring Boot web service configured to delegate all read and write operations
+to a GemFire cluster using Spring Boot and Spring Data.  
+
+In our web projects `application.properties` file, 
+we have the following line
+
+```plain
+spring.data.gemfire.pool.locators=localhost[10334]
+```
+This tells the GemFire client the location of our GemFire cluster that we created above. 
+The port `10334` is the default port, but any port can be used if configured to do so.
+
+Starting the spring boot app with GemFire can be done via
+
+```bash
+./gradlew bootRun
+```
+
+When the application starts, you should see in the Tomcat log files that the GemFire client has discovered a locator.
+
+```plain
+AutoConnectionSource discovered new locators [...:10334]
+```
+
+This means that our Spring Boot web application is connected and ready to handle requests. 
+
+## Performing Requests
+
+**The following web requests can be made to test the Cache Loader**
+
+Add a value to postgres database. This will bypass GemFire but will change a value in postgres.
+
+    psql postgres -U myuser;
+    update pgbench_tellers set filler = 'hello' where tid = 1;
+
+Retrieve value from webservice. This will invoke GemFire's CacheLoader. GemFire shouldn't know of this 
+data with the cooresponding key `1` and therefore will reach out to postgres.
+
+    curl localhost:8080/1          # should see the value hello
+
+Change value on the postgres database. Again, this will bypass GemFire but will update the value in the backing database. 
+This will setup a demonstration that GemFire is indeed caching values. 
+
+    psql postgres -U myuser;
+    update pgbench_tellers set filler = 'goodbye' where tid = 1;
+
+Retrieve value from webservice. This will still show the originally set value because the web service / GemFire was not involved. 
+
+    curl localhost:8080/1          # should still see the value hello because it is cached
+
+Clear the cache in GemFire
```

Comment:
Because we have users moving between psql, curl/web browser.gfsh, we need to indicate where to run these commands. Example, below, that command should be run in gfsh

Reply:
added